experimental/packages/otlp-transformer/src/metrics/internal.ts

```diff
@@ -129,17 +129,34 @@ function toHistogramDataPoints(
 ): IHistogramDataPoint[] {
   return metricData.dataPoints.map(dataPoint => {
     const histogram = dataPoint.value as Histogram;
-    return {
-      attributes: toAttributes(dataPoint.attributes),
-      bucketCounts: histogram.buckets.counts,
-      explicitBounds: histogram.buckets.boundaries,
-      count: histogram.count,
-      sum: histogram.sum,
-      startTimeUnixNano: hrTimeToNanoseconds(dataPoint.startTime),
-      timeUnixNano: hrTimeToNanoseconds(
-        dataPoint.endTime
-      ),
-    };
+    if(histogram.hasMinMax){
+      return {
+        attributes: toAttributes(dataPoint.attributes),
+        bucketCounts: histogram.buckets.counts,
+        explicitBounds: histogram.buckets.boundaries,
+        count: histogram.count,
+        sum: histogram.sum,
+        min: histogram.min,
+        max: histogram.max,
+        startTimeUnixNano: hrTimeToNanoseconds(dataPoint.startTime),
+        timeUnixNano: hrTimeToNanoseconds(
+          dataPoint.endTime
+        ),
+      };
+    } else {
+      return {
+        attributes: toAttributes(dataPoint.attributes),
+        bucketCounts: histogram.buckets.counts,
+        explicitBounds: histogram.buckets.boundaries,
+        count: histogram.count,
+        sum: histogram.sum,
+        startTimeUnixNano: hrTimeToNanoseconds(dataPoint.startTime),
+        timeUnixNano: hrTimeToNanoseconds(
+          dataPoint.endTime
+        ),
+      };
+    }
```

Comment:
Why do we need this if/else? If min and max are not there won't they simply be undefined?

Reply:
Thanks for catching this. For some reason I thought that this was necessary, but it is not. :sweat_smile:
I simplified this in [b23b6f1](https://github.com/open-telemetry/opentelemetry-js/pull/3032/commits/b23b6f14975841ed6ef3953bf555e0ac1a274a3c). :slightly_smiling_face: 
`min`/`max` are not undefined when they get here, though (they are `Inf` and `-1`), so I still need to check `hasMinMax` and set `undefined` explicitly.